service/controller/clusterapi/drainer.go

```diff
@@ -65,7 +68,7 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 	{
 		c := informer.Config{
 			Logger:  config.Logger,
-			Watcher: config.G8sClient.ProviderV1alpha1().AWSConfigs(""),
+			Watcher: config.CMAClient.ClusterV1alpha1().Clusters(corev1.NamespaceAll),
```

Comment:
Watching clusters here now. This breaks all resources, which I focus on get working next.